app/microservice_views.py

```diff
@@ -35,14 +35,14 @@ def create_filter_label(self, filter_name):
 
   def create_url(self, url, database_name, clear):
     if url == None:
-      url = (f'/microservices/{database_name}{clear}')
+      url = f'/microservices/{database_name}{clear}'
     else:
-      url = (f'{url}')
+      url = f'{url}'
```

Comment:
If the brackets aren't needed, this is a no-op. You can remove this entire branch, unless you need to ensure url is a string? If that's the case, pass it to `str` instead.

Reply:
Have adjusted